client/src/App.js

```diff
@@ -25,10 +25,13 @@ const App: () => React$Node = () => {
           <PickupDetail />
         </Route>
         <Route exact path="/manage-customer">
```

Comment:
these URLs need to be changed to better names too

Reply:
Dose `<Route exact path="/customer-list">`work here? But the text of the button on the landing page which directs user to this page is "Manage Customers". Should they just keep consistent?

Reply:
The goal is to make the user interface clear and intuitive, and URLs are something users can see. You can decide how to achieve that.

Reply:
I think this `<Route exact path="/manage-customer">` URL's name is ok for me, because this page is now going to show the adding customer button and run deleting customer function, also show this customers' list. What do you think?